### PR TITLE
Add govuk templates

### PR DESCRIPTION
**Pull request by tatyree**
Add in the govuk front toolkit, use high_voltage for semi-static pages, remove copy to I18N files and get rid of the accidental checkin of coverage subdir.

### CODE REVIEW

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+ @import 'elements/panels';
+ @import 'elements/forms';
+ @import 'elements/forms/form-block-labels';
+ @import 'elements/forms/form-date';
+ @import 'elements/forms/form-validation';
+ @import 'elements/breadcrumbs';
+ @import 'elements/phase-banner';
+ @import 'elements/components';
+
+/////////////////////////////
```
Comment (csutter) [inline app/assets/stylesheets/application.scss]: Might be worth taking everything below out until we need to put it back in? We might not need these styles anymore, plus they're a bit hacky.

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -1,14 +1,36 @@
...
+  <div class="header-proposition">
+    <div class="content">
+      <a href="/" id="proposition-name">Pay your tax tribunal fees</a>
+    </div>
+  </div>
+<% end %>
+
+<% content_for(:content) do %>
+  <main id="content" role="main">
+    <div class="phase-banner-alpha">
```
Comment (csutter) [inline app/views/layouts/application.html.erb]: We should change the text on this at some point (not necessarily now)

File: app/views/pages/start.html.erb [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+      <%= link_to t('.start_now') , new_case_request_path, class: 'button button-start' %>
+    </p>
+    <h2 class="heading-medium"><%= t('.before_you_start') %></h2>
+    <p>
+      <%= t('.what_you_need') %>
+    </p>
+  </div>
+
+  <div class="column-one-third">
+    <aside class="govuk-related-items" role="complementary">
```
Comment (csutter) [inline app/views/pages/start.html.erb]: Ah I see, that's where we're using the CSS. We should check if they've actually added that upstream as a component. In that case I'm fine to leave the hacky CSS in for now.

File: config/routes.rb [hunk 4]
```diff
@@ -1,3 +1,13 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  # Root route is handled by the high_voltage gem.  See
+  # config/initializers/high_voltage.rb
+
+  resources :case_requests, only: [:new, :create], path_names: { new: '' }
+
```
Comment (csutter) [inline config/routes.rb]: Do we need these routes yet?

**Comment by csutter** [comment]
See couple of comments, otherwise :+1:

File: app/views/layouts/application.html.erb (hunk 2 above)
Comment (tatyree) [inline app/views/layouts/application.html.erb]: 👍

File: config/routes.rb (hunk 4 above)
Comment (tatyree) [inline config/routes.rb]: Not yet.  I ported `routes.rb` straight.  I'll take them out.

File: app/views/pages/start.html.erb (hunk 3 above)
Comment (tatyree) [inline app/views/pages/start.html.erb]: I just did a quick search on the repo and they do not appear to have added it, yet.  I'll leave this in for now.

File: app/assets/stylesheets/application.scss (hunk 1 above)
Comment (tatyree) [inline app/assets/stylesheets/application.scss]: As per the note, below, I went ahead and left these in for now.

**Comment by csutter** [comment]
Good to squash-n-merge! :+1:

**Pull request merged**